pytorch_lightning/utilities/apply_func.py

```diff
@@ -274,7 +277,10 @@ def batch_to(data: Any) -> Any:
                 setattr(device_data, field, device_field)
             return device_data
 
-        kwargs = dict(non_blocking=True) if isinstance(data, torch.Tensor) else {}
+        kwargs = {}
+        # Don't issue non-blocking transfers to CPU. #11287
```

Comment:
There's git blame to find the associated PR number

```suggestion
        # Don't issue non-blocking transfers to CPU
```